packages/zoe/src/contracts/loan/borrow.js

```diff
@@ -48,16 +56,16 @@ export const makeBorrowInvitation = (zcf, config) => {
 
     const collateralPriceInLoanBrand = quoteAmount.value[0].amountOut;
 
-    // formula: assert collateralValue*100 >= loanWanted*mmr
+    // formula: assert collateralValue*100 >= loanWanted*mmrRatio
 
     // Calculate approximate value just for the error message if needed
-    const approxForMsg = mmr.scale(loanWanted);
+    const approxForMsg = multiplyBy(loanWanted, mmrRatio);
```

Comment:
This isn't approximate anymore, so we can just call this `requiredMargin` and reuse it below.